### PR TITLE
MM-9982 Switched from yarn to npm for the web and mobile apps

### PR DESCRIPTION
**Pull request by hmhealey**
This is mostly a simple find and replace since the two tools work similarly

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-9982

### CODE REVIEW

File: source/developer/mobile-developer-setup.rst [hunk 1]
```diff
@@ -141,14 +130,14 @@ If you need to add a new dependency to the project, it is important to add them
 JavaScript Only
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you need to add a new JavaScript dependency that is not related to React Native, **use yarn, not npm**. Be sure to save the exact version number to avoid conflicts in the future. 
+If you need to add a new JavaScript dependency that is not related to React Native, **use npm, not yarn**. Be sure to save the exact version number to avoid conflicts in the future. 
 
-	e.g. ``yarn add -E <package-name>``
+	e.g. ``npm add -E <package-name>``
```
Comment (enahum) [inline source/developer/mobile-developer-setup.rst]: this command does not exist in npm, please update

Comment (hmhealey) [reply source/developer/mobile-developer-setup.rst]: That command works fine for me

**Comment by enahum** [comment]
They probably added it

**Comment by hmhealey** [comment]
Waiting on https://github.com/mattermost/mattermost-redux/pull/460 and https://github.com/mattermost/mattermost-webapp/pull/1102

**Pull request merged**